config/default-amo.js

```diff
@@ -24,6 +24,21 @@ module.exports = {
   },
   enableTrailingSlashesMiddleware: true,
   fxaConfig: 'amo',
+
+  // This needs to be kept in sync with addons-server's SUPPORTED_NONLOCALES
+  // settings value: https://github.com/mozilla/addons-server/blob/master/src/olympia/lib/settings_base.py#L274
+  validRootUrlExceptions: [
```

Comment:
I was confused because `validRootUrlExceptions` looks like a typo. Is it not? I don't see it initialized in https://github.com/mozilla/addons-frontend/blob/44b614eba8c7692d903c7806fcd183e962da4882/config/default.js